notification-support/src/main/java/com/expediagroup/streamplatform/streamregistry/core/events/NotificationEventUtils.java

```diff
@@ -35,18 +35,39 @@
 import com.expediagroup.streamplatform.streamregistry.model.Status;
 import com.expediagroup.streamplatform.streamregistry.model.Stream;
 import com.expediagroup.streamplatform.streamregistry.model.Tag;
+import com.expediagroup.streamplatform.streamregistry.model.keys.SchemaKey;
 
 @Slf4j
 public class NotificationEventUtils {
   public static AvroKey toAvroKeyRecord(Schema schema) {
     validateSchemaKey(schema);
 
     val name = schema.getKey().getName();
+    val domainName = schema.getKey().getDomain();
```

Comment:
```suggestion
    val key = schema.getKey();
    val domainName = key.getDomain();
```

Reply:
You're right, I also modified others ones